lib/utils/shift-clay.js

```diff
@@ -1,4 +1,4 @@
-import { includes } from 'lodash';
+import _ from 'lodash';
```

Comment:
Maybe you should uhhhhhhh use `_.includes` for this later in the file? And also write tests? and maybe also test your code???????????????????

Reply:
lol this is funny...how did this even..work?

Reply:
shift-clay.js is pretty simple logic, but would be non-trivial to test. thanks for catching that syntax error tho!